src/lib.rs

```diff
@@ -356,6 +360,32 @@ impl FixedBitSet
     }
 }
 
+#[cfg(feature = "std")]
```

Comment:
These don't depend on the std feature and will work fine with core, in fact. That's just a smaller thing.

Reply:
I should remove it though if it is true. On it.